pkg/kube/pod_test.go

```diff
@@ -26,6 +26,7 @@ import (
 
 	"github.com/pkg/errors"
 	. "gopkg.in/check.v1"
+	corev1 "k8s.io/api/core/v1"
```

Comment:
Check the import below - it's the same package. I suggest we remove the old one and keep yours.

Reply:
thanks!
fixed